docs/content/docs/install/installation.md

```diff
@@ -10,6 +10,53 @@ The easiest way to install Pipelines as Code on OpenShift is with the [Red Hat O
 
 On the Openshift Pipelines Operator, the default namespace is `openshift-pipelines`.
 
+**Note:**
```

Comment:
i think it would be best to create a new page for operator instructions and move everything that belong to operator in there, it would be easier to reference and search for

Reply:
Yes that will be more clear

Or else will give details to @mramendi regarding this doc as they are working to make doc refactor and clean